### DISCUSSION TITLE
Form Fields should have a separate background theme variable?

### DISCUSSION THREAD

**Discussion by FrancescoCioria** [buildo/bento-design-system#583] [Ideas]
Currently, form fields suchs as `TextField`, share the same `--bento-backgroundColor-backgroundPrimary` CSS var used for other components such as `Card`

This means that If I set the background color of a `Card`, any form field contained in it will share the same background (appearing `transparent` to the user)

Is this wanted? How is `--bento-backgroundColor-backgroundPrimary` supposed to be used?

**Comment by gabro** [comment]
Paging @Phyele
